### DISCUSSION TITLE
Any of the Tailwind team on Mastodon?

### DISCUSSION THREAD

**Discussion by will-stone** [tailwindlabs/tailwindcss#10466] [Kind words]
I really like all the community engagement you guys do, but I unfortunately can't take Twitter any more:

- https://www.theverge.com/2023/2/2/23582615/twitter-removing-free-api-developer-apps-price-announcement
- https://www.macrumors.com/2023/01/20/twitter-bans-third-party-apps/
- https://blog.iconfactory.com/2023/01/twitterrific-end-of-an-era/

Any chance of some Mastodon love? 🐘 😄

**Comment by cannap** [comment]
try the discord server

**Reply by will-stone** [reply]
To ask the question, or as an alternative to Twitter? If the former, sure, happy to dip in and see if anyone is talking about it. As for the latter: hell no! 😱 I find Discord to be a noisy mess. I see why gamers love it, but for OSS support it's just so busy and hard to search! The threading is awful. And there's this: https://drewdevault.com/2021/12/28/Dont-use-Discord-for-FOSS.html
